it/mail/chrome/messenger/mime.properties

```diff
@@ -61,196 +61,196 @@
 
 # Resent-Comments
 ## @name MIME_MHTML_RESENT_COMMENTS
-## @loc 
+## @loc
 1001=Rispedito-Commenti
 
 # Resent-Date
 ## @name MIME_MHTML_RESENT_DATE
-## @loc 
+## @loc
 1002=Rispedito-Data
 
 # Resent-Sender
 ## @name MIME_MHTML_RESENT_SENDER
-## @loc 
+## @loc
 1003=Rispedito-Mittente
 
 # Resent-From
 ## @name MIME_MHTML_RESENT_FROM
-## @loc 
+## @loc
 1004=Rispedito-Da
 
 # Resent-To
 ## @name MIME_MHTML_RESENT_TO
-## @loc 
+## @loc
 1005=Rispedito-A
 
 # Resent-CC
 ## @name MIME_MHTML_RESENT_CC
-## @loc 
+## @loc
 # LOCALIZATION NOTE (1006): Do not translate "CC" below.
 1006=Rispedito-CC
 
 # Date
 ## @name MIME_MHTML_DATE
-## @loc 
+## @loc
 1007=Data
 
 # Sender
 ## @name MIME_MHTML_SENDER
-## @loc 
+## @loc
 1008=Mittente
 
 # From
 ## @name MIME_MHTML_FROM
-## @loc 
+## @loc
 1009=Mittente
 
 # Reply-To
 ## @name MIME_MHTML_REPLY_TO
-## @loc 
+## @loc
 1010=Rispondi-a
 
 # Organization
 ## @name MIME_MHTML_ORGANIZATION
-## @loc 
+## @loc
 1011=Organizzazione
 
 # To
 ## @name MIME_MHTML_TO
-## @loc 
+## @loc
 1012=A
 
 # CC
 ## @name MIME_MHTML_CC
-## @loc 
+## @loc
 # LOCALIZATION NOTE (1013): Do not translate "CC" below.
 1013=CC
 
 # Newsgroups
 ## @name MIME_MHTML_NEWSGROUPS
-## @loc 
-1014=Gruppi di discussione
+## @loc
+1014=Newsgroup
 
 # Followup-To
 ## @name MIME_MHTML_FOLLOWUP_TO
-## @loc 
+## @loc
 1015=Segue-in
 
 # References
 ## @name MIME_MHTML_REFERENCES
-## @loc 
+## @loc
 1016=Referenze
 
 # Name
 ## @name MIME_MHTML_NAME
-## @loc 
+## @loc
 1017=Nome
 
 # Type
 ## @name MIME_MHTML_TYPE
-## @loc 
+## @loc
 1018=Tipo
 
 # Encoding
 ## @name MIME_MHTML_ENCODING
-## @loc 
-1019=Encoding
+## @loc
+1019=Codifica
 
 # Description
 ## @name MIME_MHTML_DESCRIPTION
-## @loc 
+## @loc
 1020=Descrizione
 
 # Message ID
 ## @name MIME_MHTML_MESSAGE_ID
-## @loc 
+## @loc
 1021=ID-Messaggio
 
 # Resent Message ID
 ## @name MIME_MHTML_RESENT_MESSAGE_ID
-## @loc 
+## @loc
 1022=ID-Messaggio-Rispedito
 
 # BCC
 ## @name MIME_MHTML_BCC
-## @loc 
+## @loc
 1023=Ccn
 
 # Download Status
 ## @name MIME_MHTML_DOWNLOAD_STATUS_HEADER
-## @loc 
-1024=Download Status
+## @loc
+1024=Stato download
 
 # Download status not downloaded
 ## @name MIME_MHTML_DOWNLOAD_STATUS_NOT_DOWNLOADED
-## @loc 
-1025=Not Downloaded Inline
+## @loc
+1025=Non scaricato in linea
```

Comment:
I think that 'inline' here means attachments inside the documents (e.g. images) and not at the end of the document. Am I right? In this case I would translate:
File all'interno del documento non scaricati.
Even if I think that 'Non scaricato in linea' is a good translation in this case, where 'in linea' here doesn't mean 'online' but 'allineato'.
Please give me more context, thank you.